tasks/move-granules/tests/data/payload.json

```diff
@@ -73,18 +58,44 @@
       "name": "MOD11A1",
       "granuleId": "^MOD11A1\\.A[\\d]{7}\\.[\\S]{6}\\.006.[\\d]{13}$",
       "dataType": "MOD11A1",
-      "granuleIdExtraction": "(MOD11A1\\.(.*))\\.hdf",
       "process": "modis",
       "provider_path": "/TEST_B/Cumulus/MODIS/PDR/",
       "version": "006",
       "sampleFileName": "MOD11A1.A2017200.h19v04.006.2017201090724.hdf",
       "id": "MOD11A1"
     }
   },
-  "input": [
-    "s3://cumulus-internal/file-staging/subdir/MOD11A1.A2017200.h19v04.006.2017201090724.hdf",
-    "s3://cumulus-internal/file-staging/subdir/MOD11A1.A2017200.h19v04.006.2017201090724_1.jpg",
-    "s3://cumulus-internal/file-staging/subdir/MOD11A1.A2017200.h19v04.006.2017201090724_2.jpg",
-    "s3://cumulus-internal/file-staging/subdir/MOD11A1.A2017200.h19v04.006.2017201090724.cmr.xml"
-  ]
+  "input": {
+    "granules": [
+      {
+        "granuleId": "MOD11A1.A2017200.h19v04.006.2017201090724",
+        "files": [
+          {
+            "name": "MOD11A1.A2017200.h19v04.006.2017201090724.hdf",
+            "bucket": "cumulus-internal",
+            "filename": "s3://cumulus-internal/file-staging/subdir/MOD11A1.A2017200.h19v04.006.2017201090724.hdf",
+            "fileStagingDir": "file-staging/subdir"
+          },
+          {
+            "name": "MOD11A1.A2017200.h19v04.006.2017201090724_1.jpg",
+            "bucket": "cumulus-internal",
+            "filename": "s3://cumulus-internal/file-staging/subdir/MOD11A1.A2017200.h19v04.006.2017201090724_1.jpg",
+            "fileStagingDir": "file-staging/subdir"
+          },
+          {
+            "name": "MOD11A1.A2017200.h19v04.006.2017201090724_2.jpg",
+            "bucket": "cumulus-internal",
+            "filename": "s3://cumulus-internal/file-staging/subdir/MOD11A1.A2017200.h19v04.006.2017201090724_2.jpg",
+            "fileStagingDir": "file-staging/subdir"
+          },
+          {
+            "name": "MOD11A1.A2017200.h19v04.006.2017201090724.cmr.xml",
+            "bucket": "cumulus-internal",
+            "filename": "s3://cumulus-internal/file-staging/subdir/MOD11A1.A2017200.h19v04.006.2017201090724.cmr.xml",
+            "fileStagingDir": "file-staging/subdir"
```

Comment:
Again, to remain consistent I'd feel better if we included fileType in the input/output payload expectations to ensure we avoid clobbering them.

Reply:
👍 ea44a15bc0